version/1/3/0/pl/code_of_conduct.md

```diff
@@ -0,0 +1,24 @@
+# Kodeks postępowania współtwórców
+
+My, współtwórcy, działając dla dobra oraz otwartości naszej społeczności, ślubujemy poszanowanie dla wszystkich osób które wnoszą wartość do naszej pracy. Wartość tą wnoszą między innymi poprzez: zgłaszanie problemów, propozycji zmian, aktualizację dokumentacji, czy zgłaszanie poprawek poprzez "pull request".
+
+Jesteśmy zdeterminowani, by współpraca nad tym projektem była dla wszystkich wolna od zastraszania niezależnie od: doświadczenia; płci; identyfikacji, ekspresji i orientacji seksualnej; kalectwa; wyglądu; rozmiaru ciała; rasy; przynależności etnicznej; wieku; wyznawanej religii ani narodowości.
+
+Wśród przykładów zachowania którego nie będziemy akceptować są:
+
+* Używanie seksualnego języka i grafik,
+* Ataki osobiste,
+* "Trollowanie" i obraźliwe bądź urągające komentarze,
+* Zastraszanie, tak na forum publicznym jak i prywatne,
+* Publikowanie bez wyraźnej zgody informacji osobistych, takich jak: adres fizyczny bądź elektroniczny, 
+* Inne nieetyczne bądź nieprofesjonalne zachowania.
+
+Opiekunowie projektu zastrzegają sobie prawo do usuwania, edycji oraz odrzucania: komentarzy, commitów, kodu, wpisów na wiki oraz innych treści które łamią niniejszy Kodeks Postępowania. Ponadto zastrzegają sobie prawo do "banowania", tak tymczasowego jak i stałego, osób których zachowania uznają za: niestosowne, zastraszające, obraźliwe lub szkodliwe.
+
+Poprzez przyjęcie niniejszego Kodeksu postępowania opiekunowie ślubują, że będą cechować się uczciwością oraz konsekwentnością przy stosowaniu tych zasad w każdym aspekcie swojego działania nad tym projektem. Opiekunowie, który nie będą postępować lub wymagać od innych zachowania zgodnego z Kodeksem mogą zostać stale usunięci z zespołu współtworzącego ten projekt.
+
+Niniejszy kodeks stosuje się nie tylko w działaniach wprost związanych z tym projektem. Każdy współtwórca ślubuje zachowanie zgodne z Kodeksem kiedy tyko reprezentuje ten projekt lub jego społeczność.
+
+Przypadki obraźliwego, zastraszającego lub w inny sposób nieakceptowalnego zachowania mogą być zgłaszane do opiekuna tego projektu drogą elektroniczną pod adresem [TWÓJ ADRES EMAIL]. Wszystkie zażalenia i uwagi zostaną rozpatrzone i będą podstawą do analizy. W przypadku potwierdzenia się zarzutów zostaną podjęte odpowiednie do przewinienia środki zaradcze. Opiekunowie tego projektu są zobligowani do zachowania poufności danych osób zgłaszających incydent.
+
+Niniejszy Kodeks postępowania jest zgodny z [Contributor Covenant](http://contributor-covenant.org) w wersji 1.3.0. Oryginalny tekst można znaleźć pod adresem: [http://contributor-covenant.org/version/1/3/0/](http://contributor-covenant.org/version/1/3/0/)
```

Comment:
ślubujemy poszanowanie dla wszystkich osób które wnoszą wartość do naszej pracy. Wartość tą wnoszą między innymi poprzez: zgłaszanie problemów, propozycji zmian, aktualizację dokumentacji, czy zgłaszanie poprawek poprzez "pull request". -> ślubujemy poszanowanie dla wszystkich osób które wnoszą wkład do naszej pracy poprzez: zgłaszanie problemów, propozycji zmian, aktualizację dokumentacji, czy zgłaszanie poprawek poprzez "pull request"

Translation of the next paragraph sounds a bit akward in polish, I suggest: 
"We are committed to making participation in this project a
harassment-free experience for everyone, regardless of level of
experience, gender, gender identity and expression, sexual orientation,
disability, personal appearance, body size, race, ethnicity, age,
religion, or nationality." -> "Pragniemy, aby udział w projekcie był pozbawiony szykanowanowania ze względu na doświadczenie, płeć, orientację seksualną, stopień niepełnosprawności, wygląd, budowę ciała, kolor skóry, pochodzenie, czy wiek.